core/server/adapters/storage/LocalFileStorage.js

```diff
@@ -19,6 +19,31 @@ class LocalFileStore extends StorageBase {
         this.storagePath = config.getContentPath('images');
     }
 
+    /**
+     * Saves a buffer in the targetPath
+     * - buffer is an instance of Buffer
+     * - returns a Promise with returns the full URL to retrieve the data
```

Comment:
```suggestion
     * - returns a Promise which returns the full URL to retrieve the data
```
seems like a typo? :)